.circleci/config.yml

```diff
@@ -13,5483 +13,71 @@
+    executor: windows-gpu-prototype
     environment:
-      BUILD_ENVIRONMENT: pytorch-macos-10.13-py3-build
-    macos:
-      xcode: "9.0"
+      BUILD_ENVIRONMENT: "pytorch-win-ws2016-cuda9-cudnn7-py3"
+      SCCACHE_BUCKET: "ossci-compiler-cache"
+      CUDA_VERSION: "10"
+      USE_CUDA: "1"
     steps:
-      # See Note [Workspace for CircleCI scripts] in job-specs-setup.yml
-      - should_run_job
       - checkout
-      - run_brew_for_macos_build
       - run:
           name: Build
-          no_output_timeout: "1h"
+          no_output_timeout: "90m"
           command: |
-            set -e
-            export IN_CIRCLECI=1
-
-            # Install sccache
-            sudo curl https://s3.amazonaws.com/ossci-macos/sccache --output /usr/local/bin/sccache
-            sudo chmod +x /usr/local/bin/sccache
-            export SCCACHE_BUCKET=ossci-compiler-cache-circleci-v2
-
-            # This IAM user allows write access to S3 bucket for sccache
+            set -ex
             set +x
-            export AWS_ACCESS_KEY_ID=${CIRCLECI_AWS_ACCESS_KEY_FOR_SCCACHE_S3_BUCKET_V4}
-            export AWS_SECRET_ACCESS_KEY=${CIRCLECI_AWS_SECRET_KEY_FOR_SCCACHE_S3_BUCKET_V4}
+            export AWS_ACCESS_KEY_ID=${MINGBO_AWS_KEY_ID}
+            export AWS_SECRET_ACCESS_KEY=${MINGBO_AWS_SECRET_KEY}
             set -x
-
-            chmod a+x .jenkins/pytorch/macos-build.sh
-            unbuffer .jenkins/pytorch/macos-build.sh 2>&1 | ts
-
-            # copy with -a to preserve relative structure (e.g., symlinks), and be recursive
-            cp -a ~/project ~/workspace
-
-      - persist_to_workspace:
-          root: ~/workspace
-          paths:
-            - miniconda3
-            - project
-
-  pytorch_macos_10_13_py3_test:
-    environment:
-      BUILD_ENVIRONMENT: pytorch-macos-10.13-py3-test
-    macos:
-      xcode: "9.0"
-    steps:
-      # See Note [Workspace for CircleCI scripts] in job-specs-setup.yml
-      # This workspace also carries binaries from the build job
-      - should_run_job
-      - run_brew_for_macos_build
-      - run:
-          name: Test
-          no_output_timeout: "1h"
-          command: |
-            set -e
-            export IN_CIRCLECI=1
-
-            # copy with -a to preserve relative structure (e.g., symlinks), and be recursive
-            cp -a ~/workspace/project/. ~/project
-
-            chmod a+x .jenkins/pytorch/macos-test.sh
-            unbuffer .jenkins/pytorch/macos-test.sh 2>&1 | ts
-
-  pytorch_macos_10_13_cuda9_2_cudnn7_py3_build:
+            choco install cmake --installargs 'ADD_CMAKE_TO_PATH=User'
+            choco install -y 7zip curl awscli
+            choco upgrade -y visualstudio2019community
+            pushd ..
+            curl -O https://ossci-windows.s3.amazonaws.com/cudnn-10.1-windows10-x64-v7.6.4.38.zip
+            7z x cudnn-10.1-windows10-x64-v7.6.4.38.zip -ocudnn
+            cp -r  cudnn/cuda/* "C:/Program Files/NVIDIA GPU Computing Toolkit/CUDA/v10.1/"
```

Comment:
No cudnn pre-installed? OMG.

Reply:
Let's follow up with CircleCI on this. @mingbowan do you know how to get in touch with them?

Reply:
I can ask them to install cudnn